lib/base/error-handler.js

```diff
@@ -73,6 +73,7 @@ const log = require('npmlog');
             "INVALID_MODE" : "Please specify an option, either '--add' or '--modify'",
             "INVALID_CAPTURE_FORMAT" : "Please specify the file extension(.png, .bmp or .jpg)",
             "INVALID_COMMAND" : "This command is invalid. Please check the supported commands using ares -l",
+            "INVALID_COMBINATION" : "This options is not used with display option",
```

Comment:
Why don't you modify? 
-> "This options are not used with display option"
-> "This option cannot be used with display option"

Reply:
ok